packages/client/src/forms/countries.ts

```diff
@@ -88,6 +88,7 @@ export const countries = [
   { value: 'EST', label: messages.EST },
   { value: 'SWZ', label: messages.SWZ },
   { value: 'ETH', label: messages.ETH },
+  { value: 'FAR', label: messages.FAR },
```

Comment:
We need to have an environment variable to only add this item if the hosting URL contains farajaland